client/views/msg.tpl

```diff
@@ -6,6 +6,9 @@
 		{{#if from}}
 			{{> user_name nick=from}}
 		{{/if}}
+		{{#if nick}}
+			{{> user_name nick=nick}}
```

Comment:
Did you add this because your search results send `nick` instead of `from`? If so, can you modify it so it sends `from`?

Reply:
Yep, "from" as a field in sql database is not super handy :)
Also, I'm using "from" as time conditions from/to.
I'll see what I can do.